pkg/resolver/search.go

```diff
@@ -295,10 +281,57 @@ func formatDataMap(data map[string]interface{}) map[string]interface{} {
 			item[key] = strconv.FormatInt(int64(v), 10)
 		case map[string]interface{}:
 			item[key] = formatLabels(v)
+		case []interface{}:
+			item[key] = formatArray(v)
 		default:
 			klog.Warningf("Error formatting property with key: %+v  type: %+v\n", key, reflect.TypeOf(v))
 			continue
 		}
 	}
 	return item
 }
+
+func WhereClauseFilter(input *model.SearchInput) []exp.Expression {
```

Comment:
lowercase method name to keep private `whereClauseFilter`

Reply:
I am using that function in searchComplete file too. That is why it is public.